core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -524,6 +524,13 @@
 			'eval'                    => array('tl_class'=>'w50'),
 			'sql'                     => "int(10) unsigned NOT NULL default 0"
 		),
+		'alwaysLoadFromCache' => array
+		(
+			'label'                   => &$GLOBALS['TL_LANG']['tl_page']['alwaysLoadFromCache'],
+			'exclude'                 => true,
+			'inputType'               => 'checkbox',
+			'sql'                     => "char(1) NOT NULL default ''"
```

Comment:
add `eval => tl_class => 'w50 m12'`

Reply:
Added in d8cb92b62d099b0c7ad7d65f00835d18dc6616b2.